tools/app_interface_reporter.py

```diff
@@ -461,14 +461,16 @@ def get_repo_url(job):
 # TODO: @environ(['gitlab_pr_submitter_queue_url'])
 @gitlab_project_id
 @click.option('--reports-path', help='path to write reports')
-def main(configfile, dry_run, log_level, gitlab_project_id, reports_path):
+@click.option('--thread-pool-size', type=int, help='thread pool size')
```

Comment:
from reconcile.cli import threaded

and you can use 
@threaded()
instead

Reply:
I see! https://github.com/app-sre/qontract-reconcile/pull/1460